demos/security-trifecta/src/pages/login/LoginCard.tsx

```diff
@@ -6,17 +6,70 @@ import {
   IonCardSubtitle,
   IonCardTitle,
   IonIcon,
+  IonSelect,
+  IonSelectOption,
 } from '@ionic/react';
 import { logInOutline } from 'ionicons/icons';
+import { useEffect, useState } from 'react';
 import { useHistory } from 'react-router-dom';
 import { login } from '../../utils/auth';
+import { UnlockMode } from '../../models/UnlockMode';
+import { getUnlockMode, setUnlockMode, canUseLocking } from '../../utils/session-vault';
+import { Device } from '@ionic-enterprise/identity-vault';
+
+interface UnlockOptions {
+  mode: UnlockMode;
+  label: string;
+}
 
 const LoginCard: React.FC = () => {
   const history = useHistory();
+  const [displayLockOptions, setLockOptions] = useState<boolean>(false);
+  const [unlockModes, setUnlockModes] = useState<Array<UnlockOptions>>([]);
```

Comment:
I prefer to not nest generics like this because multiple closing `>`'s in a row gets a little messy in my opinion. Feel free to ignore this one, but now that we have a type definition to use I'd suggest writing this like: `useState<UnlockOption[]>([])`.

Reply:
I went ahead and changed it since UnlockOption[] is more readable at a glance